### DISCUSSION TITLE
SQLSTATE[23505]: Unique violation: 7 ERROR: duplicate key value violates unique constraint "users_pkey" DETAIL: Key (id)=(1) already exists.

### DISCUSSION THREAD

**Discussion by Ali-Shaikh** [laravel/framework#34345] [General]
- Laravel Version: 8.2.0
- PHP Version: 7.4.9
- Database Driver & Version: MySQL 8

### Description:
SQLSTATE[23505]: Unique violation: 7 ERROR: duplicate key value violates unique constraint "users_pkey" DETAIL: Key (id)=(1) already exists.

I have a seeder that creates three users but when I tried to register from the frontend I got this error. Shouldn't the ID be auto-generated by the DB?

**Comment by driesvints** [comment]
Converted to discussion as this is a support question.
